doc/articles/intro.md

```diff
@@ -10,17 +10,17 @@ As the Uno Platform provides all of the APIs of the complete UWP platform, any U
 
 ## Why Uno?
 
-Developing for Windows (phone, desktop, tablet, XBox), iOS (tablet and phone),  Android (tablet and phone) and WebAssembly at once can be a complex process, especially when it comes to the user interface. Each platform has its own ways of defining dynamic layouts, with some being more efficient, some more verbose, some more elegant, and some more performant than others.
```

Comment:
You changed

> Each platform has its **own ways** of defining dynamic layouts

To

> Each platform has its **ways** of defining dynamic layouts

It sounds better to say "own way" instead of "own ways" or "ways"

* Each platform has its **own way** of defining dynamic layouts

Reply:
Thanks @robloo for reviewing my pr
I made changes please check